docs/intro.md

```diff
@@ -137,7 +137,7 @@ with:
   scalability** and have **no open APIs**.
 
 You will find more details on our view over our space and competitors in the
-[Windmill compared to its peers](https://docs.windmill.dev/docs/misc/windmill_compared_to_peers) page.
+[Windmill compared to its peers](https://docs.windmill.dev/docs/misc/windmill_compared%20to%20peers/) page.
```

Comment:
@hcourdent We have links with space in it ?
like for example:  https://docs.windmill.dev/docs/misc/windmill_compared to peers